.storybook/preview.js

```diff
@@ -41,7 +42,9 @@ i18n.use(initReactI18next).init({
   },
 });
 
-window.GLOBALS = {};
+window.GLOBALS = {
+  SCRATCH_ORGS_AVAILABLE: true,
+};
```

Comment:
@dvdherron My testing showed that this var was hanging around and (potentially) impacting other stories. Given that, I'd lean toward explicitly setting it here for all stories.

Reply:
Hmm. Just curious, where did you see the variable? I called myself printing that value to the console for each component but I must've done that in the wrong place.

Reply:
In a number of other components, I put a `console.log(window.GLOBALS.SCRATCH_ORGS_AVAILABLE)` -- similar to how you were setting `window.GLOBALS.SCRATCH_ORGS_AVAILABLE` in the original component. For example:

```
const Template = (props: Props) => {
  console.log(window.GLOBALS.SCRATCH_ORGS_AVAILABLE);
  return <ProgressBar {...props} />;
};
```

When I did that, I would see `undefined` logged until after visiting the `plans/UserInfo` story, then `true` logged. I didn't spend much time on it, so I'm curious if you see something different?

Reply:
Aha, @jgerigmeyer , I misread that. I thought it logging `undefined` meant that the component just didn't have a value for `window.GLOBALS`.

Reply:
> Aha, @jgerigmeyer , I misread that. I thought it logging `undefined` meant that the component just didn't have a value for `window.GLOBALS`.

Hm, maybe I wasn't clear, because I think what you're saying is correct. I was seeing `undefined` initially on other stories (indicating that the `window.GLOBALS.SCRATCH_ORGS_AVAILABLE` value was not set), but then after visiting the `plans/UserInfo` story (where it gets set) and then going back to other stories, I would see `true` being logged (indicating that even on unrelated stories, the `window.GLOBALS.SCRATCH_ORGS_AVAILABLE` value was still set). Does that make sense? Or are you seeing something different?

Reply:
Aha, yes, that makes sense. After visiting the `plans/UserInfo` component and _returning_ to the other components they logged true. I did not notice that. Thought it was still coming from the `plans/UserInfo` component